source/mrviewerpy/MRPythonViewer.cpp

```diff
@@ -35,14 +36,14 @@ static void pythonCaptureScreenShot( MR::Viewer* viewer, const char* path )
     } );
 }
 
-static void pythonLaunch( const MR::Viewer::LaunchParams& params, const MR::ViewerSetup& setup )
+static void pythonLaunch( const MR::Viewer::LaunchParams& params, const std::shared_ptr<MR::ViewerSetup>& setup )
```

Comment:
why change to pointer? mb just take `MinimalViewerSetup` directly and hide `ViewerSetup` from user  (while it does not work)